modules/ROOT/pages/provisioning-hyperv.adoc

```diff
@@ -0,0 +1,97 @@
+= Provisioning Fedora CoreOS on Microsoft Hyper-V
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on Microsoft Hyper-V.
+
+== Prerequisites
+
+You must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+You will also need a small utility from https://github.com/containers/libhvee[libhvee] called `kvpctl`. It attaches your Ignition config to your virtual machine. Precompiled binaries can be found on the project's https://github.com/containers/libhvee/releases[releases page].
+
+=== Downloading the disk image
+
+Hyper-V disk images can be manually downloaded from the https://fedoraproject.org/coreos/download/[download page]. Be sure to decompress the image after downloading.
+
+== Booting a new VM on Microsoft Hyper-V
+
+=== Creating a virtual switch
+
```

Comment:
After launching hyper-v manger I needed to "connect to my local machine" before I was able to click on Virtual Switch Manager. Maybe we should say something here about launching Hyper-V and connecting to a host first.

Totally optional, though as I think most users will probably be able to figure it out.